pkgs/applications/video/mplayer/default.nix

```diff
@@ -199,6 +199,6 @@ stdenv.mkDerivation rec {
     homepage = "http://mplayerhq.hu";
     license = "GPL";
     maintainers = [ stdenv.lib.maintainers.eelco stdenv.lib.maintainers.urkud ];
-    platforms = stdenv.lib.platforms.linux;
+    platforms = stdenv.lib.platforms.unix;
```

Comment:
Hesitations about changing this to "unix" when the title says "Darwin support". Unix is defined to be Linux, Darwin, FreeBSD, OpenBSD, NetBSD and Illumos; as defined here: https://github.com/NixOS/nixpkgs/blob/56904d7c423f2b13b37fbd29f39bbb4b52bc7824/lib/platforms.nix#L21 . This might want to be formatted as `with stdenv.lib.platforms; linux ++ darwin ;` to conform to the header.